caboose.go

```diff
@@ -89,7 +89,7 @@ const DefaultSaturnCarRequestTimeout = 30 * time.Minute
 const defaultMaxRetries = 3
 
 // default percentage of requests to mirror for tracking how nodes perform unless overridden by MirrorFraction
-const defaultMirrorFraction = 0.01
```

Comment:
lets leave this at 0.01 for now?

we should isolate how much of the impact is due to the concurrency limitation in place currently rather than changing multiple things at once

Reply:
@willscott Yeah, makes sense. Done.